pkg/system/utimes_linux.go

```diff
@@ -3,22 +3,24 @@ package system
 import (
 	"syscall"
 	"unsafe"
+
+	"golang.org/x/sys/unix"
 )
 
 // LUtimesNano is used to change access and modification time of the specified path.
-// It's used for symbol link file because syscall.UtimesNano doesn't support a NOFOLLOW flag atm.
+// It's used for symbol link file because unix.UtimesNano doesn't support a NOFOLLOW flag atm.
 func LUtimesNano(path string, ts []syscall.Timespec) error {
-	// These are not currently available in syscall
+	// These are not currently available in unix
 	atFdCwd := -100
 	atSymLinkNoFollow := 0x100
```

Comment:
`unix.AT_FDCWD` and `unix.AT_SYMLINK_NOFOLLOW` could be used instead of these manual definitions